### DISCUSSION TITLE
How to define a custom path to the quartz.config file?

### DISCUSSION THREAD

**Discussion by CrossBound** [quartznet/quartznet#1039] [Q&A] [answered]
According to documentation here https://www.quartz-scheduler.net/documentation/quartz-3.x/configuration/reference.html#main-configuration it says you can define the "quartz.properties" system property to identify the location of the config file. I attempted to set this but I get the exception

`Quartz.SchedulerConfigException: Unknown configuration property 'quartz.properties'`

Here is my code
```csharp
                Environment.SetEnvironmentVariable("quartz.properties", Path.Combine(Program.Location, "config", "quartz.config"));
                var factory = new StdSchedulerFactory();
```

**Comment by lahma** [comment, marked as answer]
There's a problem with the documentation, the correct key is `quartz.config`, would you like to offer a PR against the docs?

Note, you can also have the program location relative configuration via `~/config/quartz.config` .

**Reply by CrossBound** [reply]
Yes, I'll try to add a PR for this when I get a chance.
